.travis.yml

```diff
@@ -70,127 +60,112 @@ rvm:
         - "pushd /tmp/beanstalkd-1.10 && make && (./beanstalkd &); popd"
     - rvm: 2.6.0
       sudo: required
       env: "GEM=activejob:integration"
+      addons:
+        postgresql: 10
+        apt:
+          packages:
+          - rabbitmq-server
       services:
         - memcached
         - redis-server
-        - rabbitmq
+        - rabbitmq-server
       before_install:
-        - sudo sed -i -e '/local.*peer/s/postgres/all/' -e 's/peer\|md5/trust/g' /etc/postgresql/*/main/pg_hba.conf
-        - "sudo sed -i 's/port = 5433/port = 5432/' /etc/postgresql/10/main/postgresql.conf"
-        - "sudo service postgresql restart 10"
         - "[ -f /tmp/beanstalkd-1.10/Makefile ] || (curl -L https://github.com/beanstalkd/beanstalkd/archive/v1.10.tar.gz | tar xz -C /tmp)"
         - "pushd /tmp/beanstalkd-1.10 && make && (./beanstalkd &); popd"
     - rvm: ruby-head
       sudo: required
       env: "GEM=activejob:integration"
+      addons:
+        postgresql: 10
+        apt:
+          packages:
+          - rabbitmq-server
       services:
         - memcached
         - redis-server
-        - rabbitmq
+        - rabbitmq-server
       before_install:
-        - sudo sed -i -e '/local.*peer/s/postgres/all/' -e 's/peer\|md5/trust/g' /etc/postgresql/*/main/pg_hba.conf
-        - "sudo sed -i 's/port = 5433/port = 5432/' /etc/postgresql/10/main/postgresql.conf"
-        - "sudo service postgresql restart 10"
+        - "rm ${BUNDLE_GEMFILE}.lock"
+        - "travis_retry gem update --system"
+        - "travis_retry gem install bundler"
         - "[ -f /tmp/beanstalkd-1.10/Makefile ] || (curl -L https://github.com/beanstalkd/beanstalkd/archive/v1.10.tar.gz | tar xz -C /tmp)"
         - "pushd /tmp/beanstalkd-1.10 && make && (./beanstalkd &); popd"
     - rvm: 2.5.3
-      env: "GEM=activerecord:mysql2"
-      sudo: required
-      before_install:
-        - "sudo mysql -e \"use mysql; update user set authentication_string='' where User='root'; update user set plugin='mysql_native_password';FLUSH PRIVILEGES;\""
-        - "sudo mysql_upgrade"
-        - "sudo service mysql restart"
-    - rvm: 2.6.0
-      env: "GEM=activerecord:mysql2"
-      sudo: required
-      before_install:
-        - "sudo mysql -e \"use mysql; update user set authentication_string='' where User='root'; update user set plugin='mysql_native_password';FLUSH PRIVILEGES;\""
-        - "sudo mysql_upgrade"
-        - "sudo service mysql restart"
-    - rvm: ruby-head
-      env: "GEM=activerecord:mysql2"
-      sudo: required
+      env:
+        - "GEM=activerecord:mysql2 MYSQL=mariadb"
       before_install:
-        - "sudo mysql -e \"use mysql; update user set authentication_string='' where User='root'; update user set plugin='mysql_native_password';FLUSH PRIVILEGES;\""
         - "sudo mysql_upgrade"
         - "sudo service mysql restart"
```

Comment:
Similar thing with "sudo service postgresql restart 10", `mariadb: 10.3` in the addons won't resolve the upgrade and the restart?